content/desktop/contributing-and-collaborating-using-github-desktop/cloning-and-forking-repositories-from-github-desktop.md

```diff
@@ -8,47 +8,38 @@ versions:
   free-pro-team: '*'
 ---
 
-### Cloning repositories
-Repositories on {% data variables.product.prodname_dotcom %} exist as remote repositories.  You can clone public repositories owned by other people. You can clone your own repository to create a local copy on your computer and sync between the two locations.
+### About local repositories
+Repositories on {% data variables.product.prodname_dotcom %} are remote repositories that you can clone or fork with {% data variables.product.prodname_desktop %} to create a local repository.
```

Comment:
Again, I know wrote this in a suggested change, but my writing process typically involves rewriting most single sentences a few times. 😂  On that note... you can always tweak suggested changes before (or after) applying them. I personally have to see everything together on staging to know if it works, which I can't do when leaving suggested changes. So I have no expectation that my changes will be applied verbatim!

Also, if you can tell before applying that the suggestion needs to be tweaked (such as the typo below) you can actually edit the comment to update the change before applying it!

```suggestion
Repositories on {% data variables.product.prodname_dotcom %} are remote repositories. You can clone or fork a repository with {% data variables.product.prodname_desktop %} to create a local repository on your computer.
```